lib/zendesk_apps_support/finders.rb

```diff
@@ -23,10 +23,12 @@ def where(arg)
     private
 
     def filter_by_arg(arg)
-      fail('More than one key-value pair found') if arg.size > 1
-      attribute, value = arg.to_a.first
-      value = value.to_s if value.is_a? Symbol
-      ->(product) { product.public_send(attribute) == value }
+      lambda do |product|
```

Comment:
Kinda strange that we name the argument `product` here, when everything else in the module has a generic name.